docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1b.md

```diff
@@ -7,6 +7,7 @@ write a file containing YAML for deploying the syncer in the guilder
 cluster.
 
 ```shell
+kubectl kcp bind apiexport root:espw:edge.kcp.io
```

Comment:
This command should not be needed, the `prep-for-syncer` plugin is not sensitive to the current working directory when you specify the IMW to be something other than `.`.

Reply:
probably true, this was desperate groping before the revelation that the docs-ecutable wa;t actually testing the current branch